ui/app/adapters/sync/destination.js

```diff
@@ -4,32 +4,25 @@
  */
 
 import ApplicationAdapter from 'vault/adapters/application';
+import { pluralize } from 'ember-inflector';
 
 export default class SyncDestinationAdapter extends ApplicationAdapter {
-  namespace = 'v1';
+  namespace = 'v1/sys';
 
-  _baseUrl() {
-    return `${this.buildURL()}/sys`;
+  pathForType(modelName) {
+    return modelName === 'sync/destination' ? pluralize(modelName) : modelName;
   }
```

Comment:
Adding `pathForType` simplifies building the url since our model directory and naming structure aligns with the paths in the endpoints.